examples/widgets/range_slider_demo.py

```diff
@@ -0,0 +1,44 @@
+"""
+======================================
+Thresholding an Image with RangeSlider
+======================================
+
+Using the RangeSlider widget to control the thresholding of an image.
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.widgets import RangeSlider
+
+fig, ax = plt.subplots()
+plt.subplots_adjust(left=0.25, bottom=0.25)
+np.random.seed(19680801)
+N = 128
+im = ax.imshow(np.random.rand(N * N).reshape(N, N))
+ax.margins(x=0)
+
+slider_ax = plt.axes([0.20, 0.1, 0.60, 0.03])
+slider = RangeSlider(slider_ax, "Threshold", 0.0, 1.0)
+
+
+def update(val):
+    im.norm.vmin = val[0]
+    im.norm.vmax = val[1]
+    fig.canvas.draw_idle()
+
+
+slider.on_changed(update)
+plt.show()
+
+#############################################################################
+#
+# ------------
+#
+# References
+# """"""""""
+#
+# The use of the following functions, methods, classes and modules is shown
+# in this example:
+
+import matplotlib
```

Comment:
How am I supposed to make flake8 not complain about this? I don't see any substantive difference from the file I borrowed how to make this reference from.

Reply:
Add an `E402` entry to `.flake8`.